rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantsService.java

```diff
@@ -17,18 +18,21 @@ public ParticipantsService(ParticipantsRepository participantsRepository) {
     this.participantsRepository = participantsRepository;
   }
 
+  @Transactional
```

Comment:
Lepiej dodać adnotację do całej klasy, czy do kilku metod z osobna?

Reply:
ja na razie dodaję na każdej z metod osobna, bo się uczę i chcę mieć świadomość tego, co się dzieje, a potem może zamienię to na adnotację na całej klasie. Na razie idę małymi krokami, żeby ogarnąć tę transakcyjność.